examples/index.json

```diff
@@ -1092,15 +1092,13 @@
       {
         "name": "Raw templates",
         "url": "raw-templates",
-        "verifyBuild": false,
-        "verifyOutput": false,
-        "disableVerificationReason": "Includes ballerinax components",
-        "disablePlayground": true,
+        "verifyBuild": true,
+        "verifyOutput": true,
         "isLearnByExample": true
       },
       {
-        "name": "Backtick templates",
-        "url": "backtick-templates",
+        "name": "String templates",
```

Comment:
I believe we need to handle redirection for this. cc @sm1990

Reply:
@heshanpadmasiri please share the old url and the new url to update the google indexing

Reply:
Old: https://ballerina.io/learn/by-example/backtick-templates/ new https://ballerina.io/learn/by-example/string-templates/

Reply:
Old: https://ballerina.io/learn/by-example/backtick-templates/ new https://ballerina.io/learn/by-example/string-templates/